client/karma.js

```diff
@@ -141,9 +141,11 @@ function Karma (socket, iframe, opener, navigator, location) {
       message += '\n\n' + error.stack
     }
 
-    // create an object with the string representation of the message to ensure all its content is properly
-    // transferred to the console log
-    message = {message: message, str: message.toString()}
+    if (typeof message !== 'string') {
```

Comment:
the logic above this line assumes that `message` is a string, so perhaps it's better to do the following:

```
    var message = messageOrEvent
    if (typeof message === "string") {
        var location = getLocation(source, lineno, colno)
        if (location !== '') {
          message += '\nat ' + location
        }
        if (error) {
          message += '\n\n' + error.stack
        }
     }
     else {
      // create an object with the string representation of the message to ensure all its content is properly
      // transferred to the console log
      message = {message: message, str: message.toString()}
    }
```

Reply:
Please take another look (the appveyor builds passed but failed to update github)